### PR TITLE
[W5.4][T11-A2] PhuaJunJie

### PR DESCRIPTION
**Pull request by Phuajunjie**

### CODE REVIEW

File: src/seedu/addressbook/commands/EditCommand.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        if (args.contains(EMAIL_PREFIX)) {
+            this.email = args.substring(args.indexOf(EMAIL_PREFIX.toString()) + 2, indexDeterminant(args));
+            args = args.substring(indexDeterminant(args)).trim();
+        }
+        if (args.contains(ADDRESS_PREFIX)) {
+            this.address = args.substring(args.indexOf(ADDRESS_PREFIX.toString()) + 2, args.length());
+        }
+    }
+
+    public int indexDeterminant(String args) {
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/commands/EditCommand.java]: Give some comments for this method. Also try to name it with getIndex... or similar.

File: src/seedu/addressbook/data/person/Phone.java [hunk 2]
```diff
@@ -36,6 +36,10 @@ public static boolean isValidPhone(String test) {
         return test.matches(PHONE_VALIDATION_REGEX);
     }
 
+    public void changePhone(String Phone){
```
Comment (OuyangDanwen) [inline src/seedu/addressbook/data/person/Phone.java]: Use the standard setter here.

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 3]
```diff
@@ -1,15 +1,33 @@
...
 import java.util.Arrays;
 import java.util.List;
 
 import org.junit.Test;
 
+import static org.junit.Assert.*;
+
 public class UtilsTest {
 
+    @Test
```
Comment (OuyangDanwen) [inline test/java/seedu/addressbook/common/UtilsTest.java]: Good test coverage!

**Comment by Phuajunjie** [comment]
@raisa2010

**Comment by raisa2010** [comment]
Code is understandable and written well. Enhancement is significant. Good job! You might want to add comments before some of the new methods though.

**Pull request closed without merging**